autoware_utils_math/include/autoware_utils_math/accumulator.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021-2024 Tier IV, Inc.
+// Copyright 2025 The Autoware Contributors
```

Comment:
Keep the publication year.
```suggestion
// Copyright 2021 The Autoware Contributors
```